.gitmodules

```diff
@@ -143,9 +143,11 @@
 [submodule "projects/preventad-registered"]
     path = projects/preventad-registered
     url = https://github.com/conpdatasets/preventad-registered.git
-[submodule "projects/calgary-campinas"]
-	path = projects/calgary-campinas
-	url = https://github.com/conpdatasets/calgary-campinas.git
 [submodule "projects/Multi-model_functionalization_of_disease-associated_PTEN_missense_mutations"]
 	path = projects/Multi-model_functionalization_of_disease-associated_PTEN_missense_mutations
 	url = https://github.com/conpdatasets/Multi-model_functionalization_of_disease-associated_PTEN_missense_mutations
+[submodule "projects/calgary-campinas"]
+	path = projects/calgary-campinas
+	url = https://github.com/conpdatasets/calgary-campinas.git
+	branch = master
```

Comment:
@emmetaobrien how does this work? If I go to https://github.com/conpdatasets/calgary-campinas.git, there are only two branches: main and git-annex. I do not see any master branch.

Reply:
That is weird, and not something I set explicitly;  I see that it is working, but am not sure how.  I will manually update the branch here to `main`.